frontend/src/components/search/SearchFilters.tsx

```diff
@@ -0,0 +1,63 @@
+"use client";
+
+import clsx from "clsx";
+
+import { useState } from "react";
+
+import ContentDisplayToggle from "src/components/ContentDisplayToggle";
+import SearchFilterAccordion, {
+  SearchFilterConfiguration,
+} from "src/components/search/SearchFilterAccordion/SearchFilterAccordion";
+import SearchOpportunityStatus from "src/components/search/SearchOpportunityStatus";
+
+export default function SearchFilters({
+  opportunityStatus,
+  filterConfigurations,
+}: {
+  opportunityStatus: Set<string>;
+  filterConfigurations: SearchFilterConfiguration[];
+}) {
+  // all filter options will be displayed on > mobile viewports
+  // on mobile, filter options will be hidden by default and can be toggled here
+  const [showFilterOptions, setShowFilterOptions] = useState(false);
+  const FilterAccordions = filterConfigurations.map(
+    ({
+      filterOptions,
+      query,
+      queryParamKey,
+      title,
+    }: SearchFilterConfiguration) => {
+      return (
+        <SearchFilterAccordion
+          filterOptions={filterOptions}
+          query={query}
+          queryParamKey={queryParamKey}
+          title={title}
+          key={title}
+        />
+      );
+    },
+  );
+  return (
+    <>
+      <div className="grid-row flex-wrap grants-search-filter-toggle">
+        <div className="grid-col-4" />
```

Comment:
What's the empty div for?

Reply:
I used flexbox with an empty div to center the toggle. If the empty div is a concern I can take a different approach

Reply:
There are utility classes you might use to do this w/o the extra markup: https://designsystem.digital.gov/utilities/flex/